<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayer</class>
 <widget class="QMainWindow" name="MediaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MediaPlayer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>File Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Value_File_Name">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_Video">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox
{
	border: none;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_Current_Time">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>00:00:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_Duration">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #666666;  /* Borda cinza */
    height: 5px;  /* Altura do groove */
    background: #333333;  /* Cor de fundo mais escura */
    
}

QSlider::handle:horizontal {
    background: #ffffff;  /* Cor do manipulador do slider */
    border: 1px solid #666666;  /* Borda cinza */
    width: 20px;  /* Largura do manipulador */
    margin: -2px -5px;  /* Ajuste de posição */
    border-radius: 6px;  
}

QSlider::add-page:horizontal {
    background: #666666;  
}

QSlider::sub-page:horizontal {
    background: #ffffff;  
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Seek_Backward">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: #ffffff;  /* Cor do texto branco */
    background-color: #444444;  /* Cor de fundo mais escura */
    border: 1px solid #666666;  /* Borda cinza */
    border-radius: 3px;
    padding: 5px 10px;
    margin: 5px;
}

QPushButton:hover {
    background-color: #555555;  /* Cor de fundo mais escura ao passar o mouse */
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Play_Pause">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: #ffffff;  /* Cor do texto branco */
    background-color: #444444;  /* Cor de fundo mais escura */
    border: 1px solid #666666;  /* Borda cinza */
    border-radius: 3px;
    padding: 5px 10px;
    margin: 5px;
}

QPushButton:hover {
    background-color: #555555;  /* Cor de fundo mais escura ao passar o mouse */
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Stop">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: #ffffff;  /* Cor do texto branco */
    background-color: #444444;  /* Cor de fundo mais escura */
    border: 1px solid #666666;  /* Borda cinza */
    border-radius: 3px;
    padding: 5px 10px;
    margin: 5px;
}

QPushButton:hover {
    background-color: #555555;  /* Cor de fundo mais escura ao passar o mouse */
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Seek_Forward">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: #ffffff;  /* Cor do texto branco */
    background-color: #444444;  /* Cor de fundo mais escura */
    border: 1px solid #666666;  /* Borda cinza */
    border-radius: 3px;
    padding: 5px 10px;
    margin: 5px;
}

QPushButton:hover {
    background-color: #555555;  /* Cor de fundo mais escura ao passar o mouse */
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_FullScreen_NormalScreen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: #ffffff;  /* Cor do texto branco */
    background-color: #444444;  /* Cor de fundo mais escura */
    border: 1px solid #666666;  /* Borda cinza */
    border-radius: 3px;
    padding: 5px 10px;
    margin: 5px;
}

QPushButton:hover {
    background-color: #555555;  /* Cor de fundo mais escura ao passar o mouse */
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Volume">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: #ffffff;  /* Cor do texto branco */
    background-color: #444444;  /* Cor de fundo mais escura */
    border: 1px solid #666666;  /* Borda cinza */
    border-radius: 3px;
    padding: 5px 10px;
    margin: 5px;
}

QPushButton:hover {
    background-color: #555555;  /* Cor de fundo mais escura ao passar o mouse */
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_Volume">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #666666;  /* Borda cinza */
    height: 5px;  /* Altura do groove */
    background: #333333;  /* Cor de fundo mais escura */
    
}

QSlider::handle:horizontal {
    background: #ffffff;  /* Cor do manipulador do slider */
    border: 1px solid #666666;  /* Borda cinza */
    width: 20px;  /* Largura do manipulador */
    margin: -2px -5px;  /* Ajuste de posição */
    border-radius: 6px;  
}

QSlider::add-page:horizontal {
    background: #666666;  
}

QSlider::sub-page:horizontal {
    background: #ffffff;  
}</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_File"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
